docs/guides/monitoring.md

```diff
@@ -0,0 +1,119 @@
+# Monitoring with Prometheus and Grafana
+
+[Prometheus](https://auth0.com/) is open source system and alerting toolkit for monitoring and alerting. It's a time series database that collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if some condition is observed to be true.
+[Grafana](https://grafana.com/oss/grafana/) is an open source visualization and analytics software. 
+
+It's a pretty common solution to use Prometheus to collect and store monitoring data, and Grafana to visualize it.
+
+Platformatic can be configured to expose Prometheus metrics: 
+
+```json
+...
+  "metrics": {
+    "port": 9091,
+    "auth": {
+      "username": "platformatic",
+      "password": "mysecret"
+    }
+  }
+...
+```
+
+In this case, we are exposing the metrics on port 9091 (defaults to `9090`), and we are using basic authentication to protect the endpoint.
+We can also specify the IP address to bind to (defaults to `0.0.0.0`). 
+Note that the metrics endpoint is exposed on a different port than the APIs. This is because if you want to test the integration running both Prometheus and Platformatic on the same host, Prometheus starts on `9090` port too.  
```

Comment:
```suggestion
Note that the metrics endpoint is exposed on a different port than the Platformatic DB APIs. This is because if you want to test the integration running both Prometheus and Platformatic on the same host, Prometheus starts on `9090` port too.  
```

Reply:
This is actually wrong @simonplend . Platformatic DB APIis on `3042`, Platformatic METRICS endpoint is on `9090`, which is the same as `Prometheus`. So Cahnging this, but specifying that we are talking about the Platformatic metric endpoint.